index.js

```diff
@@ -25,3 +25,174 @@ for (let i = 0; i < colapseBtn.length; i++) {
+    projectTags: ['html', 'javascript', 'css'], 
+    viewDemo: 'https://github.com/Ntare22/personal-portfolio-2.0',
+    viewSource: 'https://github.com/Ntare22/personal-portfolio-2.0'
+  },
+  {
+    img: './assets/landscape.png',
+    imgAlt: 'canopy',
+    projectTitle: 'Tonic',
+    projectDetails: ['CANOPY', 'Backend End Dev', '2015'],
+    projectDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    projectTags: ['html', 'javascript', 'css'],
+    viewDemo: 'https://github.com/Ntare22/personal-portfolio-2.0',
+    viewSource: 'https://github.com/Ntare22/personal-portfolio-2.0'
+  },
+  {
+    img: './assets/landscape.png',
+    imgAlt: 'canopy',
+    projectTitle: 'Jim',
+    projectDetails: ['CANOPY', 'Backend End Dev', '2015'],
+    projectDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    projectTags: ['html', 'javascript', 'css'],
+    viewDemo: 'https://github.com/Ntare22/personal-portfolio-2.0',
+    viewSource: 'https://github.com/Ntare22/personal-portfolio-2.0'
+  },
+]
+
+function modalExit() {
+  document.getElementById('modal-container').style.display = 'none';
+}
+
+function viewProject(project) {
+  // console.log('clicked on view project', project);
+  document.getElementById('modal-container').style.display = 'block';
+
+  document.getElementById('modal-title').innerHTML = project.projectTitle;
+  document.getElementById('project-det-tit').innerHTML = project.projectDetails[0];
+  document.getElementById('project-det-desc').innerHTML = project.projectDetails[1];
+  document.getElementById('project-det-yr').innerHTML = project.projectDetails[2];
+  document.getElementById('modal-img').setAttribute('src', project.img);
+  document.getElementById('project-desc').innerHTML = project.projectDesc;
+  const parentTag = document.getElementById('project-tags')
+  parentTag.innerHTML = '';
+  project.projectTags.map(tag => {
+    let newListTag = document.createElement('li');
+    newListTag.classList = 'project-tag';
+    newListTag.innerHTML = tag;
+    parentTag.appendChild(newListTag)
+  });
+  
+
+  // const title = document.getElementById('modal-title');
+  // const img = document.getElementById('modal-img')
+}
+
+function displayProjects(projects) {
+  projects.map(project => {
+    // project conteiner
+    const projectSection = document.querySelector('#works-sec');
+    const projectContainer = document.createElement('div');
+    projectContainer.classList = 'project flex';
+
+    // project img
+    const projectImg = document.createElement('img');
+    projectImg.classList = 'img-prg';
+    projectImg.setAttribute('src', project.img);
+    projectImg.setAttribute('alt', project.imgAlt);
+    
+    projectContainer.appendChild(projectImg);
+    
+    const projectInfo = document.createElement('div');
+    projectInfo.classList = 'project-info';
+    
+    //// Project Info \\\
+    // project title
```

Comment:
[OPTIONAL]
- Comments are helpful in a development environment try to keep your code clean once that you request a code review... Your code explains by itself what it do 😁.